recipe/meta.yaml

```diff
@@ -27,10 +19,7 @@ requirements:
     - make   # [unix]
     # This breaks a dependency cycle:
     # curl->libssh2->cmake->curl
-    - cmake-binary  # [not (osx and arm64)]
-    - cmake-no-system         # [osx and arm64]
-    - patch       # [not win]
-    - m2-patch    # [win]
+    - cmake-no-system
```

Comment:
is this package available for all architectures? Also for windows?

Reply:
Yes it is available for all architectures.

Reply:
ok good, then I am fine with that